Dockerfile

```diff
@@ -1,16 +1,14 @@
 FROM  python:2-stretch as builder
```

Comment:
We are using Python 3 for github actions that might be adapted in this Dockerfile -> https://github.com/symfony/symfony-docs/blob/master/.github/workflows/ci.yaml#L21

Reply:
Good catch. I will update it as it is not breaking anything.